src/resolve/resolvectl.c

```diff
@@ -1430,11 +1430,15 @@ static int status_ifindex(sd_bus *bus, int ifindex, const char *name, StatusMode
         if (mode == STATUS_DNS)
                 return status_print_strv_ifindex(ifindex, name, link_info.dns_ex ?: link_info.dns);
 
-        if (mode == STATUS_DOMAIN)
+        if (mode == STATUS_DOMAIN) {
+                strv_sort(link_info.domains);
```

Comment:
(i wonder if a smarter sort alg should be used here, i.e. one that takes common suffixes into account. i.e. a.foobar.com, z.foobar.com, quux.com should probably end up in this order, instead of a.foobar.com, quux.com, z.foobar.com. but i guess that doesn#t matter right now, stuff for another PR if anyone really cares)

Reply:
Yeah, that would be nice. But let's leave this for later.